project/src/com/company/accountservices/AccountService.java

```diff
@@ -0,0 +1,146 @@
+package com.company.accountservices;
+
+import com.company.Service;
+import com.company.accounts.Account;
+import com.company.accounts.SavingsAccount;
+import com.company.persons.Customer;
+import com.company.persons.Person;
+import com.company.personservices.CustomerService;
+import com.company.personservices.EmployeeService;
+import com.company.personservices.PersonService;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AccountService implements AccountServiceInterface{
```

Comment:
[Fix] nu cred ca ar trebui sa ai aceasta clasa, fiindca Account ar trebui sa fie abstract, si sa ai doar implementari pentru tipuri de conturi current si savings

Reply:
Am vorbit pe Teams despre acest lucru si a ramas ca pot pastra AccountService si PersonService din moment ce ma folosesc de ele in serviciile claselor derivate.

Source:
"Ma refer ca in loc sa scriu aceeasi citire a bazei Person in EmployeeService si CustomerService nu as putea sa las cum e acum si sa apelez direct citirea din PersonService chiar daca Person nu va avea instante? -> ok"